randovania/games/dread/logic_database/Dairon.txt

```diff
@@ -285,6 +285,14 @@ Extra - asset_id: collision_camera_004
               All of the following:
                   Flash Shift
```

Comment:
Unrelated but this could also use a flash upgrade.

Reply:
Out of scope.